include/zephyr/bluetooth/bluetooth.h

```diff
@@ -231,6 +231,8 @@ int bt_enable(bt_ready_cb_t cb);
  *
  * Disable Bluetooth. Can't be called before bt_enable has completed.
  *
+ * After this API is called, all the previously configured identities are cleared.
+ *
```

Comment:
Aren't IDs store in settings if `CONFIG_BT_SETTINGS=y`? 

Wouldn't it only be if `CONFIG_BT_SETTINGS=n` that this is true?